CHANGELOG.md

```diff
@@ -20,13 +20,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
-- [#73](https://github.com/nf-core/epitopeprediction/pull/73)   - Add support for the non-free `NetMHC` tool family including `NetMHC 4.0`, `NetMHCpan 4.0`, `NetMHCII 2.2`, and `NetMHCIIpan 3.1`
-- [#83](https://github.com/nf-core/epitopeprediction/pull/83)   - Add option for threshold customization
+- [#73](https://github.com/nf-core/epitopeprediction/pull/73) - Add support for the non-free `NetMHC` tool family including `NetMHC 4.0`, `NetMHCpan 4.0`, `NetMHCII 2.2`, and `NetMHCIIpan 3.1`
+- [#83](https://github.com/nf-core/epitopeprediction/pull/83) - Add option for threshold customization
 - [#101](https://github.com/nf-core/epitopeprediction/pull/101) - Add local modules for DSL2 conversion
 
 ### `Changed`
 
-- [#107](https://github.com/nf-core/epitopeprediction/pull/107)  - Merge previous template updates up to `v2.1`
+- [#107](https://github.com/nf-core/epitopeprediction/pull/107) - Merge previous template updates up to `v2.1`
 - [#110](https://github.com/nf-core/epitopeprediction/pull/110), [#113](https://github.com/nf-core/epitopeprediction/pull/113) - Port pipeline to Nextflow DSL2 syntax
 - [#114](https://github.com/nf-core/epitopeprediction/pull/114) - Update `python 2.7` to `python 3.8.9` in `split_peptides.nf` and `merge_json.nf`.
 - [#117](https://github.com/nf-core/epitopeprediction/pull/117) - Bump minimal NXF version to `21.10.4`
```

Comment:
21.10.3 is the version used in .github/workflows/ci.yml and specified in nextflow.config, so that seems the minimal version